README.md

```diff
@@ -81,7 +86,7 @@ You need to mount all required files and folders to run the container:
 docker run \
     -it \
     --name eelbot \
-    -v <full/path/to/EelbotDB.db>:/app/EelbotDB.db \
+    -v <full/path/to/eelbot.db>:/app/eelbot.db \
     -v <full/path/to/config.ini>:/app/config.ini \
     -v <full/path/to/pics/folder>:/app/pics \
```

Comment:
I would update `pics` to be `images` to be consistent with the naming everywhere else.